regression/scripts/noise_fidelity_one/propeller_noise.py

```diff
@@ -77,18 +81,21 @@ def main():
     conditions.noise.microphone_locations        = mic_locations
     
     # Run Propeller model 
-    F, Q, P, Cp , noise_data , etap                        = prop.spin(conditions)   
+    F, Q, P, Cp , noise_data , etap                        = prop.spin(conditions)
```

Comment:
I thought about this a lot! I wanted to change this so much!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

I have a plan for how I'd like to see the data structures.

But I didn't want to change too much in this PR since I wasn't the one who put this in. Let's have a talk at the next meeting.